src/js/ui/DomHandler.js

```diff
@@ -44,7 +76,7 @@ class DomHandler {
             optionsHtml +=
                 "<div class=\"form-check\">\n"
                 + "<input class=\"form-check-input\" type=\"radio\" name=\"decisionsRadio\" id=\"" + opt
-                + "\"> <label class=\"form-check-label\" for=\"" + opt + "\">" + opt + "</label>"
+                + "\" value='"+opt+"'> <label class=\"form-check-label\" for=\"" + opt + "\">" + opt + "</label>"
```

Comment:
also irgendwie erscheint mit das etwas komisch, wenn hier html zeug zusammengebastelt wird... zumal das auch echt unleserlich ist mit den ganzen escape chars... aber ich hab halt auch absolut kein Dunst ob das in js einfach immer so ist oder ob es da irgendwie eine elegantere lösung gibt

Reply:
AFAIK gibt es da ohne vollständiges JS Framework keine wirklich schönere Lösung...